chaos_wrapper.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+function rollback() {
+    # avoid commits from the same faulty PR
+    rb_commit=$(git log --author Chaos --format=format:%H | tail -n +2 | head -n 1)
```

Comment:
After thinking a bit more about it I realised that it's still not bulletproof 
"attacker" can just make a commit like this: `git commit --author "Chaos"` 

I'm afraid we can't solely rely on git for this ;(